source/contentRecog/recogUi.py

```diff
@@ -37,11 +39,65 @@ class RecogResultNVDAObject(cursorManager.CursorManager, NVDAObjects.window.Wind
 	name = _("Result")
 	treeInterceptor = None
 
-	def __init__(self, result=None, obj=None):

```

Comment:
not taking result as an input parameter may be an API breaking change

Reply:
You're correct. I'll fix that.